src/boot/efi/chid.h

```diff
@@ -0,0 +1,6 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "efi.h"
+
+EFI_STATUS hwid_match(const void *dtb_buffer, size_t dtb_length);
```

Comment:
But you call this `chid_match` now, no?

Reply:
You're right, must be an oversight. Not sure which name to keep though.